src/atoms/Layout/index.js

```diff
@@ -4,6 +4,7 @@ import cx from 'classnames';
 import { processChildren } from './utils';
 import styles from './layout.module.scss';
 import Col from './Col';
+import LayoutGrayBox from './LayoutGrayBox';
```

Comment:
I think you can just do:

`export LayoutGrayBox from './LayoutGrayBox';` and not update the bottom.